pkg/testframework/framework.go

```diff
@@ -159,15 +172,27 @@ func IsResourceAvailable(errChan chan error, client *Clientset, resource string)
 	errChan <- err
 }
 
+func ConsoleResourcesUnavailable(client *Clientset) error {
```

Comment:
Ah, yeah we already have `IS` in `IsResourceUnavailable` lol.  Oh well, naming is hard.